scripts/notebook.ipynb

```diff
@@ -84,17 +84,18 @@
     "    \"pathParameters\": {\"topic_name\": \"edla.change.topic\"},\n",
     "    \"headers\": {\"bearer\": jwtToken},\n",
     "    \"body\": json.dumps({\n",
-    "        \"app_id_snow\": \"app-1234\",\n",
-    "        \"data_definition_id\": \"data-def-5678\",\n",
-    "        \"environment\": \"DEV\",\n",
-    "        \"format\": \"JSON\",\n",
-    "        \"guid\": \"550e8400-e29b-41d4-a716-446655440000\",\n",
-    "        \"location\": \"s3://data-lake/customer_data\",\n",
-    "        \"operation\": \"CREATE\",\n",
-    "        \"schema_link\": \"https://schema-registry.company.com/schemas/data-def-5678\",\n",
-    "        \"source_app\": \"DataIngestionApp\",\n",
-    "        \"timestamp_event\": 1657896543210\n",
-    "    })\n",
+    "        \"event_id\": \"JupyterEventId\",\n",
+    "        \"tenant_id\": \"JupyterTenantId\",\n",
+    "        \"source_app\": \"JupyterSrc\",\n",
+    "        \"source_app_version\": \"v2024-10-17\",\n",
+    "        \"environment\": \"JupyterEnv\",\n",
+    "        \"timestamp_event\": 20241017,\n",
```

Comment:
Just to stress this is is epoch milli
```suggestion
    "        \"timestamp_event\": 1729602770000,\n",
```

Reply:
I mean, that is sample script, might as well be there 1234 for easy visual check message arrived... but sure why not